shared/common-adapters/error-profile.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import React from 'react'
+import Box from './box'
+import Text from './text'
+import {globalColors, globalMargins} from '../styles/style-guide'
+
+export type Props = {
+  error: string
+}
+
+export default function ErrorLoadingProfile ({error}: Props) {
```

Comment:
That sounds good to me, this is the bare minimum so users don't get into a broken state.

We can iterate from there.